src/components/pages/Chat/EmptyFeed/index.tsx

```diff
@@ -13,16 +15,45 @@ const CreatedAt = styled.span`
     margin-bottom: 2rem;
 `
 
-const UnstyledEmptyFeed = ({ className, roomCreatedAt }: Props) => {
+const UnstyledEmptyFeed = ({ className }: Props) => {
+    const [roomCreatedAt, setRoomCreatedAt] = useState(0)
```

Comment:
`0` gives you false expectations. ;) Let's adjust for `undefined` which is more accurate at this stage.